WMass/python/plotter/ccFiles/systHistHelpers.cc

```diff
@@ -15,7 +15,14 @@ T mirrorHist(T& nominal, T& alternate, T& mirrored) {
         double yM = (yA > 0 && y0 > 0) ? y0*y0/yA : 0;
         //std::cout << "Here yM is " << yM << " y0 is " << y0 << " ratio is " << y0/yA << std::endl;
         mirrored.SetBinContent(b, yM);
-        mirrored.SetBinError(b, alternate.GetBinError(b)*y0/yA);
```

Comment:
But is alternate.GetBinError(b) set anywhere? I think that if we keep "alternate.GetBinError(b)*y0/yA" we should actually use nominal.GetBinError(b)*y0/yA. Or, if alternative and nominal have the same uncertainties because of cloning the alternative from nominal, then so should mirror.

Reply:
This assumes that the alternate hist was built from a proper estimation, if will have proper unc if it's produced by weights for example. The only reason I can see that you'd want the uncertainty of the alternate hist is to see if there is a high stat uncertainty somehow, and for that the alternate unc is more useful.

But fine, you have it for one or the other, I don't really care if you set it to nominal.GetBinError(b)